Net/CMakeLists.txt

```diff
@@ -18,12 +18,21 @@ set_target_properties( Net
 target_link_libraries(Net PUBLIC Poco::Foundation)
 # Windows and WindowsCE need additional libraries
 if(WIN32)
-    target_link_libraries(Net PUBLIC "iphlpapi.lib")
+    target_link_libraries(Net PUBLIC "iphlpapi")
     if(WINCE)
-        target_link_libraries(Net PUBLIC "ws2.lib")
+        target_link_libraries(Net PUBLIC "ws2")
     else()    
-        target_link_libraries(Net PUBLIC "ws2_32.lib")
-    endif()    
+        target_link_libraries(Net PUBLIC "ws2_32")
+    endif()
+
+    if(MINGW) # Needed by WinAPI in MinGW
+        string(REGEX MATCH "^[0-9]+\\.[0-9]+" WIN_VERSION ${CMAKE_SYSTEM_VERSION})
```

Comment:
In cmake it is possible to verify version numbers with `VERSION_LESS` etc. See https://cmake.org/cmake/help/latest/command/if.html?highlight=version_less

Reply:
It doesn't check or verify version, it calculate it from version like `6.0.141241` to `0x0600`.
* Take first two numbers `6` and `0`, so we get `6.0`
* It take major version, so we get `6`
* It take minor version, so we get `0`
* Transform these two numbers to version like WinAPI. `256 * 6 + 0 = 1536` or `0x0600`
If windows version differ, WinAPI will be different too.

Reply:
ok